_includes/tutorials/splitting/kstreams/markup/dev/make-topology.adoc

```diff
@@ -1,5 +1,7 @@
-Then create the following file at `src/main/java/io/confluent/developer/SplitStream.java`. Notice the `buildTopology` method, which uses the Kafka Streams DSL. The `branch` method takes varargs of predicate functions over each record's key and value. `branch` returns an array of `KStream`, with the order of the array matching the the ordering of the predicates. If no predicates are matched, the event gets dropped from further processing. Notice the last predicate, which simply returns `true`. This acts as an "else" statement to catch all events that don't match the other predicates.
+Then create the following file at `src/main/java/io/confluent/developer/SplitStream.java`. Notice the `buildTopology` method, which uses the Kafka Streams DSL. The `split` operation takes varargs of predicate functions over each record's key and value, and `Branched` creates a branch for messages that match the predicate and returns this in order to facilitate method chaining.  If no predicates are matched, the event gets dropped from further processing, but in this case, notice the last predicate, which simply returns `true`. This acts as an "else" statement to catch all events that don't match the other predicates.
```

Comment:
I think this change resulted in some incorrect language:

* The `split` function is overloaded to take no arguments or an instance of `Named` (https://kafka.apache.org/28/javadoc/org/apache/kafka/streams/kstream/KStream.html#split())
* `Branched` -> `branch` and it takes the predicate function as the parameter

Reply:
Updated in latest commit